src/components/Form/Form.vue

```diff
@@ -1,22 +1,25 @@
 <template>
   <form class="form">
     <input class="button" type="submit" name="nearestLocation" value="Nearest Location" v-on:click.prevent="findNearestLocation">
-    <select class="dropdown" name="" @change="selectState">
+    <label for="states">States:</label>
+    <select class="dropdown" id="states" @change="selectState">
```

Comment:
I'd have to look at whether it would actually work to apply the same way to `<select>` tags in Vue,
just noting that we may be able to apply an `aria-label` attribute to that tag itself instead of the separate `<label>` tag if we feel the placeholders are effective enough for the visual descriptors and don't necessarily need the labels displayed.